examples/config_tiny_llama.yaml

```diff
@@ -69,7 +69,7 @@ optimizer:
   weight_decay: 0.01
   zero_stage: 0
 parallelism:
-  dp: 16
+  dp: 4
```

Comment:
For one node with 8 GPUs and `tp=2`, I don't think it's possible to have `dp=16` since then the world size is 32, not 8